components/backup/src/endpoint.rs

```diff
@@ -24,6 +24,9 @@ use tokio_threadpool::{Builder as ThreadPoolBuilder, ThreadPool};
 use crate::metrics::*;
 use crate::*;
 
+const WORKER_TAKE_RANGE: usize = 5;
```

Comment:
Can it be multiple of 2?

Reply:
Oh, it was random picked. What are the benefit if it is multiple of 2?

Reply:
It may reduce memory consumption by trying to algin data structure.

https://en.wikipedia.org/wiki/Data_structure_alignment

Reply:
Oh, thanks!